langkit/support/langkit_support-token_data_handlers.ads

```diff
@@ -113,6 +109,8 @@ package Langkit_Support.Token_Data_Handlers is
 
    package Index_Vectors is new Langkit_Support.Vectors (Positive);
 
+   type Tab_Stop_Access is access constant Positive;
```

Comment:
When creating a TDH, the tab stop info comes from the context and is read-only: why is it materialized using an access type here, instead of a simple positive value? Also, this lacks a docstring

Reply:
I think I didn't catch that the info was read-only in the context, and wanted to give a way for it to work if the tab stop was dynamically changed.

Reply:
Anyway, will change it!